docs/assets/fhir-resource-ingestion-digestion-diagrams/bfd-pipeline-rda-ingestion.plantuml

```diff
@@ -0,0 +1,30 @@
+@startuml
+title BFD-Pipeline-RDA-GRPC Ingestion of Partially Adjudicated Claims Data Sequence
+participant "rda-pipeline-source"
+participant "rda-streaming-api"
+database "rda-db"
+participant "rda-pipeline-sink"
+database "bfd-db"
+note over "rda-streaming-api": Runs every 5 min in prod \n 1 hr in test
+"rda-pipeline-source" -> "rda-streaming-api": Smoke test to determine connectivity
+"rda-pipeline-source" -> "rda-db": Smoke test to determine connectivity
+"rda-streaming-api" --> "rda-pipeline-source"  
+"rda-db" --> "rda-pipeline-source"
+"rda-pipeline-source" -> "rda-db": Dead Letter Queue (DLQ) Processing
+"rda-db" --> "rda-pipeline-source"
+"rda-pipeline-source" -> "rda-streaming-api": Run 'DLQGrpcRdaSource' job if needed
+"rda-streaming-api" -> "rda-pipeline-source"
+"rda-pipeline-source" -> "rda-streaming-api": GetVersion: stream of current API version
+"rda-streaming-api" --> "rda-pipeline-source"
+"rda-pipeline-source" -> "rda-db": Get starting sequence number to process claims
+"rda-db" --> "rda-pipeline-source"
```

Comment:
RDA pipeline queries a table in the `rda` schema of the BFD dataase to obtain the starting sequence number value.  Is `rda-db` here meant to refer to the BFD `rda` schema?  Or to the RDA API's own internal database?

If the former then the arrow for writing claims to the database should be changed to reflect that.  If the latter this should be changed to point to `bfd-db` instead.

Reply:
Ah yes, that's what I meant.